client/src/partials/journal/modals/trialBalanceMain.footer.html

```diff
@@ -0,0 +1,22 @@
+
+
+<button
+  id="cancelID"
+  type="button"
+  class="btn btn-default"
+  ng-click="TrialBalanceMainFooterCtrl.cancel()">
+    {{ "FORM.BUTTONS.CANCEL" | translate }}
+</button>
+
+<button
+  ng-if="TrialBalanceMainFooterCtrl.state.current.data.checked"
+  type="button"
+  ng-class="{
+    'btn btn-danger' : TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasError,
+    'btn btn-warning': TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasWarning,
+    'btn btn-primary': TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasSuccess}"
+  ng-disabled="TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasError"
+  ng-click="TrialBalanceMainFooterCtrl.submit()"
+  data-method="submit">
```

Comment:
It would be really nice if this could have a loading indicator on it as well.  For thousands of records, I assume the Trial Balance will take some time to compute.

It doesn't have to be done in this PR, but should be done eventually.